src/content/api/tapable.md

```diff
@@ -8,13 +8,13 @@ contributors:
   - e-cloud
 ---
 
-[Tapable](https://github.com/webpack/tapable) is small library that allows you to add and apply plugins to a javascript module. It can be inherited or mixed in to other modules. It is similar to NodeJS's `EventEmitter` class, focusing on custom event emission and manipulation. However, in addition to this, `Tapable` allows you to have access to the "emittee" or "producer" of the event through callbacks arguments.
+[Tapable](https://github.com/webpack/tapable) a is small library that allows you to add and apply plugins to a javascript module. It can be inherited or mixed in to other modules. It is similar to NodeJS's `EventEmitter` class, focusing on custom event emission and manipulation. However, in addition to this, `Tapable` allows you to have access to the "emittee" or "producer" of the event through callbacks arguments.
```

Comment:
Hmm, I think you meant `is a` instead of `a is`?

Reply:
@skipjack whoops you're absolutely right, thanks for the correction!

Reply:
i will take a look at the rest of these later when I'm home, fix them up so we can move forward. Quick question though, I made these corrections through the GitHub UI. If I want to update this PR to handle your feedback, should that be done on a local branch on my computer? I don't see anywhere on GitHub to revise this specific PR.

Newbie here :)

Thanks again Greg!

Reply:
> i will take a look at the rest of these later when I'm home, fix them up so we can move forward.

Great, no rush.

> If I want to update this PR to handle your feedback, should that be done on a local branch on my computer? I don't see anywhere on GitHub to revise this specific PR.

If you go to the `Files changed` tab,  you should see an "Edit" icon on the file that will allow you to edit it again through the UI. You could also `git clone` your fork and push to that `patch-1` branch but you should be able to just make the changes through the interface.

![image](https://user-images.githubusercontent.com/8988697/31842948-9383f356-b5be-11e7-803e-cd29149f0b5f.png)

You should see that same pencil icon on the right.